core/src/main/java/org/apache/lucene/queryparser/classic/MapperQueryParser.java

```diff
@@ -85,14 +85,14 @@
 
     private String quoteFieldSuffix;
 
-    public MapperQueryParser(QueryParseContext parseContext) {
+    public MapperQueryParser(QueryShardContext context) {
         super(null, null);
-        this.parseContext = parseContext;
+        this.context = context;
     }
 
-    public MapperQueryParser(QueryParserSettings settings, QueryParseContext parseContext) {
+    public MapperQueryParser(QueryParserSettings settings, QueryShardContext context) {
```

Comment:
this constructor is unused, both in our branch and in master. We can remove it I think